submissions/description/server_security_misconfiguration/insecure_ssl/certificate_error/template.md

```diff
@@ -0,0 +1,20 @@
+# Certificate Error
+
+## Overview of the Vulnerability
+
+When a web page is loaded, the browser makes a series of checks against the website’s Secure Socket Layer (SSL) certificate. When these checks fail, the browser blocks the webpage from loading and issues a certificate error. An attacker can leverage this information to identify what caused the certificate error and execute a Person-in-The-Middle (PitM) attack to view and modify information that passes between the vulnerable endpoints.
+
+## Business Impact
+
+Certificate errors can lead to reputational damage for the business due to a loss in confidence and trust by users who attempt to connect to the webpage and receive a certificate error.
+
+## Steps to Reproduce
+
+1. Navigate to the following URL to find a certificate error is thrown by the browser: {{URL}}
+1. Sensitive information can be discovered by {{action}}
```

Comment:
What is sensitive about a certificate error?

Reply:
I'm not sure. It was in the previous template's steps to reproduce so I left it in there to help researchers convey impact.  I wasn't sure if I was missing something with certificate errors where an attacker could discover sensitive info. 

I can remove this step if there's nothing aside from viewing the certificate error needed.

Reply:
Likely that's the best option here, just to remove it